.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing Guidelines
+
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## Pull Requests
+
+Please ensure these poitns are followed during PR:
+
+- Ensure a clean commit history - make one commit per feature and squash relevant fixes if needed
```

Comment:
We should always squash the fixes to a single commit, so should read "and squash relevant fixes"